### DISCUSSION TITLE
Errors with Running the visualization

### DISCUSSION THREAD

**Discussion by ssomani10** [WAM2layers/WAM2layers#220] [Q&A] [answered]
I was running through flood case example and had no trouble with the backtrack process.  However, I when I tried to use the visualization command,  using a cut and paste from read the docs I got the following error. Please let me know if I should repost in the discussion forum.  Thanks.

<img width="1340" alt="Visualization Error" src="https://github.com/WAM2layers/WAM2layers/assets/8172915/860f5ed9-365e-4dca-9613-9677476f0ac7">

**Comment by ssomani10** [comment]
Just to followup with that, I have attached the screenshot with me running the backtrack program and then obtaining an error when trying the visualization.  I'm not sure if it's a Mac problem, or a computer code since one works, but the other one doesn't.  If you are not having that issue, then please close it out and I will repost on the discussion forum an try it on a Linux platform.

<img width="1621" alt="WAM2Layers Visualization" src="https://github.com/WAM2layers/WAM2layers/assets/8172915/eb3ec6e9-cb9a-477b-a9d5-ae25cb117b63">

**Comment by Peter9192** [comment]
Looking at the error message (`No such file or directory`), it seems that there is some issue with the paths where it's looking for output. I would start by trying to look at the output folder (`floodcase 202107/output_data`). Is just this one file missing, or is there no output at all?

**Comment by ruudvdent** [comment]
I'll be transferring this to the Q&A discussion forum

**Comment by ssomani10** [comment]
There's no files that are missing.  I have attached a screenshot to the output_data directory.  All files are there from July 2-July15 and the files sizes are not null.  Let me know if there is anything else you need.  Totally guessing and wondering if it's an issue with the  mac and that would be solved via using Linux.   

<img width="1362" alt="Output Directory" src="https://github.com/WAM2layers/WAM2layers/assets/8172915/6d49f117-179b-4b4e-a3f1-55cd7b456a2d">

**Reply by Peter9192** [reply, marked as answer]
Look more closely. The code is trying to load the data for the first of july, you have files starting at the second. I suspect that the model is not writing the last output, this is buggy behaviour that should be fixed. But for now, a fix could be to change the start date in your config file such that the analysis won't look for the first of july.

**Reply by Peter9192** [reply]
Opened an issue here: https://github.com/WAM2layers/WAM2layers/issues/222

**Comment by ssomani10** [comment]
Thanks., that did the trick  I should have confirmed the dates in the YAML file.
